client/src/servicesURLs.js

```diff
@@ -0,0 +1,3 @@
+export const MENUITEM_SERVICE_URL = "https://localhost:44347/api/MenuItem";
```

Comment:
It would be better to move this file to `constants` subfolder where later we can place other constants